src/cowboy_telemetry_h.erl

```diff
@@ -0,0 +1,57 @@
+-module(cowboy_telemetry_h).
+-behavior(cowboy_stream).
+
+-export([init/3]).
+-export([data/4]).
+-export([info/3]).
+-export([terminate/3]).
+-export([early_error/5]).
+
+init(StreamID, Req, Opts) ->
+  SystemTime = erlang:system_time(),
+  StartTime = erlang:monotonic_time(),
+  telemetry:execute(
+    [cowboy, stream_handler, start],
```

Comment:
`stream_handler` is still a bit of an implementation detail, and it's not true that all stream handler events are emitted, so I'm thinking that `[cowboy, request, start|stop|exception]` would be ideal...